packages/@lightningjs/ui-components/src/components/ListItem/ListItem.mdx

```diff
@@ -65,12 +61,10 @@ class Basic extends lng.Component {
 
 `ListItem` has the same properties as the ones listed in [Button](?path=/docs/components-button--basic) in addition to a couple listed below:
 
-| name           | type    | default   | description                                                                           |
-| -------------- | ------- | --------- | ------------------------------------------------------------------------------------- |
-| description    | string  | undefined | description text                                                                      |
-| prefixLogo     | string  | undefined | Logo to be placed to the left of the title and description                            |
-| shouldCollapse | boolean | false     | flag that if `true`, hides the title when `ListItem` is in unfocused or disabled mode |
-| suffixLogo     | string  | undefined | Logo to be placed to the right of the title and description                           |
+<ArgTypes
+  of={ListItemStories.ListItem}
+  exclude={['suffix', 'prefix', 'title', 'mode']}
+/>
```

Comment:
Everything in the prior table is getting pulled from the `argTypes` in the component's story